app/templates/_base_page.html

```diff
@@ -1,7 +1,11 @@
 {% extends "govuk/govuk_template.html" %}
 
 {% block head %}
-  {% include "_custom_meta.html" %}
+  {% with custom_dimensions = [
+    {"data_id": 10, "data_value": current_user.role }
```

Comment:
Potentially the association of a particular dimension id (i.e. 10) with a particular field (i.e. user.role) should also be centralized. Whatever happens, we can't ever redefine that in another app.

Do all our apps put `current_user` into the template? If so, maybe this can literally be lifted into toolkit template rather than passed in.

Reply:
I think it's in the context for all apps, however after checking with Ash, we don't want to set this particular dimension if the user isn't logged in (i.e. where `current_user.role == None`). We could do a null check on `data-value` in the toolkit but it feels like that logic should be at app level. 

Also this means we can reuse the `custom-dimensions.html` template for other dimensions set on specific URLs (e.g. the saved search status on the Buyer FE), without having to worry about duplicating the user role dimension on the same page. 

I agree that having a centralised list of dimensions and their IDs is sensible in the long run, but it's probably out of scope for this story?

Reply:
You're very much relying here on the fact that the briefs app is logged-in-only. In general we might be redirecting people to any app (well, particularly in the case of buyers, they might be redirected to a page on the buyer app, e.g. G-Cloud pages which are available both logged-in and logged-out).

In which case you will anyway have to put an `if` into the template to do the check for the not-logged-in case -- and that would be the same in the other apps, at least if they needed to handle both logged in and not.

It definitely feels to me like it might be the right thing to do to centralize this from the off?